src/main/java/hudson/plugins/analysis/core/PluginDescriptor.java

```diff
@@ -122,21 +128,6 @@ public PluginDescriptor(final Class<? extends Publisher> clazz) {
     }
 
     @Override
-    public Publisher newInstance(final StaplerRequest req, final JSONObject formData)
-            throws hudson.model.Descriptor.FormException {
-        JSONObject converted;
-        if (formData.containsKey(CONFIGURATION_SECTION_KEY)) {
-            JSONObject old = formData.getJSONObject(CONFIGURATION_SECTION_KEY);
-            formData.put(CONFIGURATION_SECTION_KEY, convertHierarchicalFormData(old));
-            converted = formData;
-        }
-        else {
-            converted = convertHierarchicalFormData(formData);
-        }
-        return super.newInstance(req, converted);
-    }
```

Comment:
Did you analyze what this override was for, and confirm that it was safe to delete?